managed/models/job_helpers.go

```diff
@@ -82,7 +82,6 @@ func FindJobs(q *reform.Querier, filters JobsFilter) ([]*Job, error) {
 		crossJoin = true
 		andConds = append(andConds, "value ->> 'restore_id' = "+q.Placeholder(idx))
 		args = append(args, filters.RestoreID)
-		idx++
```

Comment:
I think it's better to leave this increment, but comment it. So when we copy this part next time we don't forget about increment.